tests/tools/plugins/Makefile.inc

```diff
@@ -67,6 +67,9 @@ tools_plugins_ssl_verify_test_la_SOURCES = tools/plugins/ssl_verify_test.cc
 noinst_LTLIBRARIES += tools/plugins/ssntxnorder_verify.la
 tools_plugins_ssntxnorder_verify_la_SOURCES = tools/plugins/ssntxnorder_verify.cc
 
+noinst_LTLIBRARIES += tools/plugins/hook_tunnel_plugin.la
```

Comment:
Please also add this to the CMake build.  Suggested change in tests/tools/plugins/CMakeLists.txt: `add_autest_plugin(hook_tunnel_plugin hook_tunnel_plugin.cc)`

Reply:
Added